<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>707</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="matrixB_Label">
          <property name="text">
           <string>矩阵B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="readB_Button">
          <property name="text">
           <string>从.txt文件读入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearB_Button">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="matrixB_textBrowser"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="matrixA_Label">
          <property name="text">
           <string>矩阵A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="readA_Button">
          <property name="text">
           <string>从.txt文件读入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearA_Button">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="matrixA_textBrowser">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="status_label">
      <property name="text">
       <string>状态栏</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="operationMode_comboBox">
          <item>
           <property name="text">
            <string>请选择运算模式</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>加法</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>减法</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>乘法</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="execute_Button">
          <property name="text">
           <string>执行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_Button">
          <property name="text">
           <string>保存结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearResult_pushButton">
          <property name="text">
           <string>清空结果</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="result_textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
